app/src/main/res/layout/options_display.xml

```diff
@@ -90,6 +90,12 @@
                     app:options="@array/developer_options_msaa"
                     app:values="@array/developer_options_msaa_mode_values" />
 
+                <com.igalia.wolvic.ui.views.settings.RadioGroupSetting
+                    android:id="@+id/windows_size"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:description="Window's&#10;default size" />
```

Comment:
One final detail, this needs to be a string resource. Let's use "Default window size".